megamek/src/megamek/client/ui/swing/ClientGUI.java

```diff
@@ -390,28 +408,29 @@ public void setMiniReportDisplayDialog(final MiniReportDisplayDialog miniReportD
     /**
      * Try to load the "bing" sound clip.
      */
-    private void loadSoundClip() {
-        if (GUIP.getSoundBingFilename() == null) {
-            return;
+    private Clip loadSoundClip(String filename) {
+        Clip clip = null;
+
+        if (filename == null) {
+            return null;
         }
-        final File file = new File(GUIP.getSoundBingFilename());
+        final File file = new File(filename);
         if (!file.exists()) {
-            LogManager.getLogger().error(Messages.getString("ClientGUI.failedToLoadAudioFile") + " " + GUIP.getSoundBingFilename());
-            return;
+            LogManager.getLogger().error(Messages.getString("ClientGUI.failedToLoadAudioFile") + " " + filename);
+            return null;
         }
 
         try {
-            if (bingClip != null) {
-                bingClip.close();
-            }
-            bingClip = AudioSystem.getClip();
+            clip = AudioSystem.getClip();
             try (AudioInputStream ais = AudioSystem.getAudioInputStream(file)) {
-                bingClip.open(ais);
+                clip.open(ais);
             }
         } catch (Exception ex) {
             LogManager.getLogger().error("", ex);
-            bingClip = null;
+            clip = null;
         }
+
+        return clip;
```

Comment:
I would recommend returning the clip at the end of the try, and just returning null in the catch. It's slightly better performing.